src/automata/StreamSubscriptionRequesterBase.cpp

```diff
@@ -113,8 +113,8 @@ void StreamSubscriptionRequesterBase::onNextFrame(Frame_ERROR& frame) {
       break;
     case State::REQUESTED:
       state_ = State::CLOSED;
-      Base::onError(
-          std::runtime_error(frame.data_->moveToFbString().toStdString()));
+      Base::onError(std::runtime_error(
+          frame.payload_.data->moveToFbString().toStdString()));
```

Comment:
will add moveDataToString. I want to make sure that the reader is aware of the destruction of the payload.data field.